common/src/main/java/org/opensearch/ml/common/transport/upload_chunk/MLRegisterModelMetaInput.java

```diff
@@ -29,20 +29,23 @@ public class MLRegisterModelMetaInput implements ToXContentObject, Writeable{
 
     public static final String FUNCTION_NAME_FIELD = "function_name";
     public static final String MODEL_NAME_FIELD = "name"; //mandatory
-    public static final String DESCRIPTION_FIELD = "description";
+    public static final String DESCRIPTION_FIELD = "description"; //optional
+
+    public static final String VERSION_FIELD = "version";
```

Comment:
Per my understand, If model group id provided, version is still optional, if model group id is not provided, user need to input version. Is that correct?

Reply:
Yes, thats correct